### ISSUE TITLE
Add pagination support or helping out to connect with `less -R` and have colors survive

### ISSUE THREAD

**Issue by fperez** [kracekumar/jut#11]
I'm finding that even if I use `less -R`, when paginating jut's output I lose color highlights, not sure why... But given that your typical notebook document is long-ish, I would suggest adding pagination support less-Style, if it's not too hard... 

Alternatively, figure out a way if possible to make it so coloring survives through `less`, which would save you from having to reimplement less's features (I'm sure if you add pagination, people will then want search, regexes, etc, and you'd end up having to reinvent less itself :)

**Comment by kracekumar** [comment]
Yes, this is due to rich way of handling the color. I haven't gotten my head around it. Will look at the ways to handle the color.

Regarding the search, I intend to support filter the cells by type(https://github.com/kracekumar/jut/issues/4) and by search term(https://github.com/kracekumar/jut/issues/2).

thanks for the suggestion :-)

**Comment by punchagan** [comment]
@fperez There was a --force-colors option added in 8783d212b4beeeb675ecfd6dea6d82d27229115d which should make `jut` play well with `less -R`. :)
